pkgs/by-name/ae/aerospace/package.nix

```
@@ -55,7 +55,7 @@ stdenv.mkDerivation {
     homepage = "https://github.com/nikitabobko/AeroSpace";
     description = "i3-like tiling window manager for macOS";
     platforms = lib.platforms.darwin;
-    maintainers = with lib.maintainers; [ alexandru0-dev ];
+    maintainers = with lib.maintainers; [ alexandru0-dev ok-nick ];
```

you should run `nixfmt` from `nixfmt-rfc-style` package on the patch
```suggestion
    maintainers = with lib.maintainers; [
      alexandru0-dev
      ok-nick
    ];
```
I have no issue for you to be added as a maintainer (idk how useful that would be for this package as it's minimal tbh, maybe a from src package?)
I also enabled the releases notification on aerospace a couple of months ago so i don't miss anything, but I was asleep when I received it
I would have hoped that r-ryantm would help more so I can also merge them myself but it's finding updates with some delays (you can visit https://r.ryantm.com/log/aerospace/)

---

I like the notifications when this package is updated in nixpkgs.

---

Wait.. so you only want the notifications or do you want to maintain this package?
Because if it's the latter, it's ok

Otherwise I would like to request from you to actually remove yourself from the maintainers attribute

---

Right.. I sent this PR just for the notification…